graphql/admin/endpoints_ee.go

```diff
@@ -71,6 +71,18 @@ const adminTypes = `
 		"""
 		backupNum: Int
```

Comment:
We can clear up this wording (i.e. number of _backups_ within series)

Reply:
done